pkg/asset/machines/aws/worker.go

```diff
@@ -23,60 +30,70 @@ type MachineConfig struct {
 	Machine     types.AWSMachinePoolPlatform
 }
 
-// WorkerMachineSetTmpl is template for worker machineset.
-var WorkerMachineSetTmpl = template.Must(template.New("aws-worker-machineset").Parse(`
-apiVersion: cluster.k8s.io/v1alpha1
-kind: MachineSet
+// WorkerMachineSetsTmpl is template for worker machinesets.
+var WorkerMachineSetsTmpl = template.Must(template.New("aws-worker-machinesets").Parse(`
```

Comment:
At some point it would be good to revisit using templates here vs building up a MachineSet object and serializing it. I know you mentioned 1.9 apimachinery vendoring problems from the cluster-api repo, but they seem to be up to 1.11.2 now, which might help: https://github.com/kubernetes-sigs/cluster-api/blob/master/Gopkg.lock#L643

This would be helpful because we may need to share some code here. Installer will generate and create the initial machine sets, but Hive's API type will need to be synced over to the cluster over time as machine pools are scaled up, added, removed, flipped to autoscaling, etc. It seems like we might want some common code for converting a list of machine pools to a list of expected machine sets we can reconcile.

Reply:
https://jira.coreos.com/browse/CORS-875 and https://jira.coreos.com/browse/CORS-876